.github/workflows/release.yml

```diff
@@ -14,29 +14,31 @@ jobs:
   main:
     # ignore all-contributors PRs
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
+    name: Node ${{ matrix.node }}, Svelte ${{ matrix.svelte }}
     strategy:
       matrix:
-        node: [16, 18, 20]
+        node: ['16', '18', '20']
```

Comment:
We can probably drop Node v16 from the CI matrix, but I'd rather leave that for a breaking change release. I'm pretty sure there's one on the horizon for the rerender stuff, at least

Reply:
Yeah, we can drop it when the reports get too cumbersome. Till then, more data is always nice.